container-scan/action.yaml

```diff
@@ -1,152 +1,74 @@
-name: "Security Scan Composite Action"
-description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+name: Container Security Scan
+description: Scans container images for vulnerabilities using Lacework
 
 inputs:
-  dockerhub-user:
-    required: false
-    default: ""
-    description: username for dockerhub
-  dockerhub-password:
-    required: false
-    default: ""
-    description: password for dockerhub
-  docker-config-file:
-    required: false
-    description: Path to the docker config file (defaults to .docker-config.json) Must contain imageName, may contain dockerfile
-    default: .docker-config.json
-  github-token:
-    required: true
-    description: GitHub token
-  lw-account-name:
-    required: true
-    description: Lacework account name
-  lw-access-token:
-    required: true
-    description: Lacework access token
   image-name:
     required: false
     description: Docker image name
   image-tag:
     required: true
     description: Docker image tag
-  image-platform:
```

Comment:
Sorry, I missed this too, but this is an input that you should keep and pass through I think as well